server/filters/grammar.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+type AOrAn = 'a' | 'an';
+export function aOrAn(string: string): AOrAn {
+  return 'aeiou'.indexOf(string.charAt(0).toLowerCase()) !== -1 ? 'an' : 'a';
```

Comment:
just a nitpick, this will be incorrect for words like "universal" and "useful"

Reply:
When we hit that I might introduce a lib, Thanks for pointing it out!

Reply:
yeah – I this would be tricky to get right – it's the vowel sound, rather than the vowel, so a load of abbreviations that start without vowels would fail too.

Reply:
If it gets any more complicated, we'll shift it to content creators.